jsonrpsee/Cargo.toml

```diff
@@ -24,7 +24,7 @@ jsonrpsee-types = { path = "../types", version = "0.6.1", package = "jsonrpsee-t
 [features]
 client-ws-transport = ["jsonrpsee-client-transport/ws", "jsonrpsee-client-transport/tls"]
 client-ws-transport-no-tls = ["jsonrpsee-client-transport/ws"]
-core-client = ["jsonrpsee-core/client"]
+core-client = ["jsonrpsee-core/async-client"]
```

Comment:
Maybe `async-client` is a better name for the feature, so it's consistent and we avoid the ambiguous "core"?

Reply:
agree.